src/functions.js

```diff
@@ -128,7 +221,13 @@ function factorial( int ) {
 
   // calculate and return the factorial of int
   // note: factorial of 0 is 1
-}
+  var factorial = int =>(int===0) ? 1 : int*factorial(int-1);
```

Comment:
With this solution, whilst it worked, you've broken out of the function structure, and created a variable on the global scope. For the purposes of this exercise, it's not the best answer, thought you should be able to wrap it in a function to bring it in line with the way you've completed the other functions.